picard/formats/vorbis.py

```diff
@@ -139,7 +139,7 @@ def _load(self, filename):
             for value in values:
                 value = value.rstrip('\0')
                 name = origname
-                if name == "date" or name == "originaldate":
+                if name == "date" or name == "originaldate" or name == "releasedate":
```

Comment:
It might be cleaner if you used a set here similar to your change in `edittagdialog.py`.  Something like:

```python
if name in {'date', 'originaldate', releasedate'}:
```

Same comment applies to your change in Line 280.

Reply:
Done, thanks!